tests/network/vmi_multus.go

```diff
@@ -741,6 +761,12 @@ var _ = Describe("[Serial]SRIOV", func() {
 			}
 		}
 
+		checkEnvVariablesInPod := func(vmi *v1.VirtualMachineInstance, networkName string) {
+			vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
+			By("checking KUBEVIRT_RESOURCE_NAME_<networkName> variable is defined in pod")
+			Expect(validatePodKubevirtResourceName(virtClient, vmiPod, networkName, sriovResourceName)).To(Succeed())
+		}
+
```

Comment:
Why is this function is a lambda instead of regular function?
I think its better if we could have this function to return an error instead
so the usage would be as before `Expect(checkEnvVariablesInPod(...)).To(Succeed())`.

Reply:
Partially done. I keep the function a lambda, as this check is only done in this one `Context` (if I'm looking correctly). Also, there are already similar lambdas defined.
Fixed the helper to return error.

Reply:
Thanks, its better now